api/users/views.py

```diff
@@ -18,11 +21,13 @@ class UserMixin(object):
 
     def get_user(self, check_permissions=True):
         obj = get_object_or_404(User, self.kwargs[self.node_lookup_url_kwarg])
-        if check_permissions:
+
+        if check_permissions & obj.is_disabled is False:
```

Comment:
But to answer the broader question specifically, we do not want any user that is inactive to be accessible through these endpoints.